definitions/npm/stringify-object_v3.x.x/test_stringify-object_3.x.x.js

```diff
@@ -0,0 +1,111 @@
+// @flow
```

Comment:
Thanks for writing tests; please organize them with `describe` and `it` blocks.

Reply:
Done.